.github/workflows/workflow.yml

```diff
@@ -4,51 +4,53 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
please, extract this into separate PR to make each PR targeted to single change

Reply:
done